ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -298,6 +298,9 @@ extension BrowserViewController: WKNavigationDelegate {
     }
 
     let isPrivateBrowsing = privateBrowsingManager.isPrivateBrowsing
+    tab?.isNewNavigation =
+      navigationAction.navigationType != WKNavigationType.backForward
+      && navigationAction.navigationType != WKNavigationType.reload
```

Comment:
Is this an acceptable way to determine if its a new navigation? There are other types here

Reply:
Chromium has the same check using:
```
  return ui::PageTransitionIsNewNavigation(
      navigation_handle->GetPageTransition());
```

Reply:
yes, and `PageTransitionIsNewNavigation` checks back/forward and reload transitions in implementation:
```

bool PageTransitionIsNewNavigation(PageTransition type) {
  return (type & PAGE_TRANSITION_FORWARD_BACK) == 0 &&
      !PageTransitionCoreTypeIs(type, PAGE_TRANSITION_RELOAD);
}
```